articles/google-play-in-app-reviews/index.md

```diff
@@ -0,0 +1,250 @@
+When a user is interacting with your application, you always want to get feedback. Feedback will give you insight into what your users think about your application.
+
+In this guide, we'll learn how to implement the "rate this app" functionality within your android application using Google Core in-app review API.
+
+In the old days, a user would have to open the Google Play store to write a review. In some fashion, you would use a fallback review dialog with some "call to action" buttons to nudge the user to review your application. If the user agrees, the button redirects to the Play store so that the user can write a review about your application.
+
+This approach does not promote a user good experience and can potentially ruin your application's reputation among users. To make matters worse, this method doesn’t establish communication between the Play store and your application; it will keep popping up even after a user leaves a review. This type of call to action will force your user to leave your application. And you don’t want such a scenario, as there's a chance that a user may not reopen your application once redirected to the play store.
+
+Google announced an [in-app review API](https://developer.android.com/guide/playcore/in-app-review) that prompts users to submit Play store ratings and reviews without leaving the application and without opening the Play store either. Users' reviews/ratings will eventually be displayed in the Google Play store. This encourages users to give feedback on your application.
+
+![image](/engineering-education/google-play-in-app-reviews/in-app-review-flow.jpg)
+
+***[Image source.](https://developer.android.com/guide/playcore/in-app-review)***
+
+This is how the in-app review API looks compared to the old fallback dialog.
+
+![image](/engineering-education/google-play-in-app-reviews/in-app-dialog-and-fallback-dialog.png)
+
+Typically, this API concept is a better way to ask users for a review than the fallback rate dialog.
+
+### Prerequisites
+This guide assumes you have prior knowledge of Android application development using Android Studio.
+
+### Why you should ask your users for reviews
+Reviews play a marketing role in your application. They are a crucial part of securing more downloads because they can convince a new user whether they should download the application. There's a good chance a new user will check reviews and ratings to decide which application to try. The reviews give the experiences of previous users.
+
+Reviews also form a line of communication between your users and the application, as they tell you what to improve in your application. Potentially, a user could catch a bug while interacting with your application. Feedback lets you know that something might not be working well, giving you a nudge to fix a bug.
```

Comment:
Is it establishing communication between the user and the application or it is between the user and the developer???